### PR TITLE
[bitnami/nginx] add strategy value for the deployment

### LINKED ISSUE #9054: nginx strategy type
### Name and Version

bitnami/nginx 9.7.6

### What is the problem this feature will solve?

there is no way to set/define strategy type for the deployment

### What is the feature you are proposing to solve the problem?

provide an ability to change/set strategy type for the deployment

### What alternatives have you considered?

custom chart

### PR DESCRIPTION
**Pull request by messiahUA**
**Description of the change**

Add a value to set deployment strategy.

**Benefits**

I would be possible to change the strategy for the deployment.

**Possible drawbacks**

None.

**Applicable issues**

  - fixes #9054

**Additional information**

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by messiahUA** [comment]
not sure why version bump check fails:
"Error: Detected changes in charts without version bump in Chart.yaml."
I've bumped the version...

File: bitnami/nginx/README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ The command removes all the Kubernetes components associated with the chart and
 | Name                                    | Description                                                                               | Value   |
 | --------------------------------------- | ----------------------------------------------------------------------------------------- | ------- |
 | `replicaCount`                          | Number of NGINX replicas to deploy                                                        | `1`     |
+| `strategy`                              | Strategy for the deployment                                                               | `{}`    |
```
Comment (juan131) [inline bitnami/nginx/README.md:100]: Please rename this parameter to "updateStrategy". We're using this name on every chart regardless it's a deployment or a statefulset.

File: bitnami/nginx/values.yaml [hunk 2]
```diff
@@ -98,6 +98,16 @@ extraEnvVarsSecret: ""
...
+## @param strategy Strategy for the deployment
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## e.g:
+## strategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+##
+strategy: {}
```
Comment (juan131) [inline bitnami/nginx/values.yaml:110]: Please use this instead:
```suggestion
## @param updateStrategy.type NGINX deployment strategy type
## @param updateStrategy.rollingUpdate NGINX deployment rolling update configuration parameters
## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
##
updateStrategy:
  type: RollingUpdate
  rollingUpdate: {}
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for your PR @messiahUA ! Please take a look to my comments when you have a chacne.

**Comment by messiahUA** [comment]
@juan131 done, but I still don't understand why it complains about the version bump

**Review by juan131 — APPROVED** [review]
LGTM

**Comment by juan131** [comment]
> but I still don't understand why it complains about the version bump

To be honest, me neither @messiahUA. Probably it's because you're creating the PR from the "master" branch of you fork instead of creating a branch in your fork but I'm not very sure about this. 

I'll proceed and merge your PR regardless that test

**Pull request merged**